_includes/footer.html

```diff
@@ -31,18 +31,37 @@
       <div class="suggestion-text">{{{text}}}</div>
     </div>
   </div>
-  {% endraw %}
+  {% endraw %}-
   </script>
-
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.4/js/bootstrap.min.js"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/anchor-js/1.1.1/anchor.min.js"></script>
-  <script src="{{ site.url }}/js/app/search.js"></script>
+  <script src="https://cdn.jsdelivr.net/g/lodash@3.10.1,hogan.js@3.0.2,algoliasearch@3.8.1,autocomplete.js@0.12.0(autocomplete.min.js)" defer></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap.native/2.0.15/bootstrap-native.min.js" defer></script>
+  <script src="/js/app/anchor.min.js" defer></script>
```

Comment:
I think the perf improvement for custom anchor is minimal, so I'd rather take a small hit in perf, but keep dependency intact. Unless you think this is critical.

Reply:
It causes around 700ms to load the additional font in case of using the original one. If you are ok with that then I'll revert this commit too.
![image](https://user-images.githubusercontent.com/6231516/31409873-05357e4e-ae16-11e7-8b3d-894384c47790.png)

Reply:
Sorry, I messed up, glyphicons it's own eslint site fonts. So it looks fine because of https://github.com/eslint/eslint.github.io/pull/418/files#diff-110b203ee900658cc6a1d3772a89c033R17
I'll use anchorjs together from cdn.